deployments/gpu_plugin/overlays/nfd_labeled_nodes/add-nodeselector-intel-gpu.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-gpu-plugin
+spec:
+  template:
+    spec:
+      nodeSelector:
+        feature.node.kubernetes.io/pci-0300_8086.present: "true"
```

Comment:
one problem occured to me: the planned QAT setting would conflict with this. Meaning: we cannot deploy QAT plugin with the same NFD pci setting (`class_vendor` vs `device_vendor`) than GPU... we'll need to re-think this a bit.